packages/default/scss/listbox/_theme.scss

```diff
@@ -1,4 +1,7 @@
 @include exports('listbox/theme') {
+    $listbox-drag-clue-bg: $list-item-selected-bg !default;
+    $listbox-text: $list-item-selected-text !default;
```

Comment:
The variable name is misleading. I would expect to change the text color of the listbox widget, not the dragclue.